test/unit/tessel.js

```diff
@@ -35,17 +35,17 @@ exports['Tessel (endConnection)'] = {
     test.done();
   },
 
-  closeConnection: function(test) {
-    test.expect(3);
+  // closeConnection: function(test) {
+  //   test.expect(3);
 
-    var length = process._events.SIGINT.length;
-    var tessel = new Tessel(this.mockConnection);
+  //   var length = process._events.SIGINT.length;
+  //   var tessel = new Tessel(this.mockConnection);
 
-    test.equal(process._events.SIGINT.length, length + 1);
-    tessel.close();
+  //   test.equal(process._events.SIGINT.length, length + 1);
+  //   tessel.close();
 
-    test.equal(process._events.SIGINT.length, length);
-    test.equal(this.end.callCount, 0);
-    test.done();
-  },
+  //   test.equal(process._events.SIGINT.length, length);
+  //   test.equal(this.end.callCount, 0);
+  //   test.done();
+  // },
```

Comment:
Was this intentional? What's the issue? Keep in mind: checking the events is not ideal, but was the quickest way to confirm that an explicit `tessel.close();` had removed the event. Now that I'm revisiting it, I think this should be changed to:

```js
  closeConnection: function(test) {
    test.expect(2);

    var processremoveListener = sinon.stub(process, 'removeListener');
    var tessel = new Tessel(this.mockConnection);

    tessel.close();

    test.equal(processremoveListener.callCount, 1);
    test.equal(this.end.callCount, 0);

    processremoveListener.restore();
    test.done();
  },
```


I just tried that locally and it passes and doesn't make me want to vomit ;)